src/common.jl

```diff
@@ -116,3 +116,133 @@ function my_mul_mt(
     end
     return y
 end
+
+function make_dc_powerflowdata(
+    sys,
+    time_steps,
+    timestep_names,
+    power_network_matrix,
+    aux_network_matrix,
+    n_buses,
+    n_branches,
+    bus_lookup,
+    branch_lookup,
+    temp_bus_map,
+    valid_ix,
+)
+    branch_types = Vector{DataType}(undef, length(branch_lookup))
+    for (ix, b) in enumerate(PNM.get_ac_branches(sys))
+        branch_types[ix] = typeof(b)
+    end
+    bus_reactivepower_bounds = Vector{Vector{Float64}}()
+    timestep_map = Dict(zip([i for i in 1:time_steps], timestep_names))
+    neighbors = Vector{Set{Int}}()
+    return make_powerflowdata(
+        sys,
+        time_steps,
+        power_network_matrix,
+        aux_network_matrix,
+        n_buses,
+        n_branches,
+        bus_lookup,
+        branch_lookup,
+        temp_bus_map,
+        branch_types,
+        bus_reactivepower_bounds,
+        timestep_map,
+        valid_ix,
+        neighbors,
+    )
+end
+
+function make_powerflowdata(
+    sys,
+    time_steps,
+    power_network_matrix,
+    aux_network_matrix,
+    n_buses,
+    n_branches,
+    bus_lookup,
+    branch_lookup,
+    temp_bus_map,
+    branch_types,
+    bus_reactivepower_bounds,
+    timestep_map,
+    valid_ix,
+    neighbors,
+)
+    # TODO: bus_type might need to also be a Matrix since the type can change for a particular scenario
```

Comment:
Lets do this change right away

Reply:
In private discussion with @jd-lara, the resolution to this (old) TODO is to wait until we implement more multi-period AC power flow infrastructure.

Reply:
Logged at https://github.com/NREL-Sienna/PowerFlows.jl/issues/52